pagopa/io-app
# fix(IDPay): [IODPAY-102] Add service info header in IDPay onboarding screen

##  ⚠️ This PR depends on https://github.com/pagopa/io-app/pull/4535

## Short description
This PR adds service info header (name, organization name and logo) in initiative onboarding screen.

https://user-images.githubusercontent.com/6160324/231755834-dab9d125-09a2-4cd5-964a-67b2ae51a6de.mp4

## List of changes proposed in this pull request
- Added `OnboardingServiceHeader`
- Refactoring of `InitiativeDetailsScreen` to handle correct data

## How to test
With the `io-dev-server`, navigate to the IDPay Onboarding playground and start a new onboarding. Check that service header is correctly displayed.
